packages/realm-server/migrations/1726671342065_backfill-realm-owners.js

```diff
@@ -0,0 +1,111 @@
+exports.up = (pgm) => {
+  switch (process.env.REALM_SENTRY_ENVIRONMENT) {
+    case 'staging':
+      pgm.sql(
+        `INSERT INTO realm_user_permissions (realm_url, username, read, write, realm_owner)
+         VALUES
+           ('https://realms-staging.stack.cards/seed/', '@seed_realm:stack.cards', true, true, true),
+           ('https://realms-staging.stack.cards/experiments/', '@experiments_realm:stack.cards', true, true, true),
+           ('https://realms-staging.stack.cards/experiments/', '*', true, false, false),
+           ('https://realms-staging.stack.cards/experiments/', 'users', true, true, false),
+           ('https://realms-staging.stack.cards/catalog/', '@catalog_realm:stack.cards', true, true, true),
+           ('https://realms-staging.stack.cards/catalog/', '*', true, false, false),
+           ('https://cardstack.com/base/', '@base_realm:stack.cards', true, true, true),
+           ('https://cardstack.com/base/', '*', true, false, false)`,
+      );
+      break;
+    case 'production':
+      pgm.sql(
+        `INSERT INTO realm_user_permissions (realm_url, username, read, write, realm_owner)
+         VALUES
+           ('https://app.boxel.ai/seed/', '@seed_realm:boxel.ai', true, true, true),
+           ('https://app.boxel.ai/experiments/', '@experiments_realm:boxel.ai', true, true, true),
+           ('https://app.boxel.ai/experiments/', '*', true, false, false),
+           ('https://app.boxel.ai/experiments/', 'users', true, true, false),
+           ('https://app.boxel.ai/catalog/', '@catalog_realm:boxel.ai', true, true, true),
+           ('https://app.boxel.ai/catalog/', '*', true, false, false),
+           ('https://cardstack.com/base/', '@base_realm:boxel.ai', true, true, true),
+           ('https://cardstack.com/base/', '*', true, false, false)`,
+      );
+      break;
```

Comment:
Maybe these need to be upserts? I got a uniqueness constraint error deploying this in production.

Reply:
sure thing